pyproject.toml

```diff
@@ -216,9 +216,6 @@ exclude_also = [
 ]
 skip_covered = true
 
-[tool.distutils.bdist_wheel]
-py_limited_api = "cp310"
```

Comment:
Same comment as on the feature flag in Cargo.toml (although I'm not sure if we really still need this for abi3 wheels to work).